gsva_tools/run_gsva.xml

```diff
@@ -0,0 +1,277 @@
+<!-- Author: Jonah Hocum -->
+<!-- Email: hocumj@ohsu.edu -->
+<!-- Description: A simple galaxy wrapper tool to run GSVA. -->
+<!-- TODO (hocumj 7/15/2019): Consider moving some code to a macro.xml file. -->
+<!-- TODO (hocumj 7/15/2019): Handle different delimiters. -->
+<!-- TODO (hocumj 7/15/2019): Implement try/except error handling. -->
+<!-- TODO (hocumj 7/15/2019): Provide default gene sets. -->
+<!-- TODO (hocumj 7/15/2019): Allow combining gene sets from multiple files. -->
+<!-- TODO (hocumj 7/15/2019): Add preprocessing step to handle missing Ensembl ID versions. E.g., ENSG00000171223.5 compared to ENSG00000171223 -->
+<!-- TODO (hocumj 7/15/2019): Implement some tests for running with non-default parameters. -->
+
+<tool id="run_gsva" name="Run GSVA:" version="0.1.0">
```

Comment:
Suggest changing `Run GSVA:` to either `GSVA` or `Gene Set Enrichment Analysis`. Current name can look awkward when referring to the tool in places other than the tool panel (such as the admin panel).